### PR TITLE
Fix no scale

### PR DESCRIPTION
**Pull request by jeantuffier**
So there is a problem with the calculation of top left picture coordinate : when scale factor is exactly 1, meaning the user hasn't zoom, the value is not correct. If you want more explanation come by where I seat I can show you more details

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,4 @@
 *.iml
 .gradle/
 .idea/
-app/app.iml
-imagecrop/imagecrop.iml
```
Comment (sindrenm) [inline .gitignore:6]: Can we please keep these ignored?

**Comment by sindrenm** [comment]
I really don't understand why this works, nor why the scaling has anything to do with whether or not the offset screws up. Care to write down a small explanation about why scaling messes with the values? :smiley:

File: .gitignore (hunk 1 above)
Comment (jeantuffier) [inline .gitignore:6]: I remove them because it has no effect in my cli... I still have them in the "not staged".

**Comment by jeantuffier** [comment]
So the problem comes from this line in `getCroppedPicture()` : 

``` java
int x = Math.round(cropRect.left) - Math.round(left); 
```

it returns 470 when the scale factor is one instead of 420 (for s5 devices). The error com from using `Math.round(cropRect.left)` instead of using the actual calculation which is `Math.round((middle.x - cropSize/2)/mScaleFactor)` or just `Math.round(middle.x - cropSize/2)` since the scale factor is 1.

The thing is I have no idea why they don't return the same value

**Comment by jeantuffier** [comment]
Also I updated the library on bintray, win can import 0.3.3 now

**Comment by sindrenm** [comment]
Did you update the BinTray version before this was merged?

**Comment by jeantuffier** [comment]
Done now

**Comment by sindrenm** [comment]
Right, okay.

**Pull request merged**